framework/components/ema-content-slider/ema-content-slider.scss

```diff
@@ -0,0 +1,28 @@
+.ema-content-slider {
+  margin: 0;
+}
+
+@media (min-width: map-get($ecl-grid-breakpoints, 'md')) {
+  .ema-content-slider {
+    display: grid;
+    grid-template-columns: 60% 40%;
+  }
+  .ema-content-slider .ecl-list-item {
+    align-items: center;
+    grid-column-start: 2;
+    &:not([aria-hidden='true']) {
+      display: flex;
+    }
+  }
+  .ema-content-slider .ema-content-slider__list-item--highlight {
```

Comment:
selected item should be somehow highlighted in the list (there is a border on mockups for instance)